components/mediation-ui/mediators-ui/org.wso2.carbon.mediator.call.ui/src/main/java/org/wso2/carbon/mediator/call/CallMediator.java

```diff
@@ -54,6 +67,9 @@ public OMElement serialize(OMElement parent) {
         if (activeEndpoint != null) {
             call.addChild(EndpointSerializer.getElementFromEndpoint(activeEndpoint));
         }
+        if (blocking==true) {

```

Comment:
The suggested change is fixed in the latest code in the master branch.
if (blocking) {